app/views/issues/index.html.erb

```diff
@@ -34,6 +34,9 @@
   </div>
 </div>
 
+<div class="container container-1 mt-5">
```

Comment:
CSS comments:

- the container is not aligned left/right with the container below. The style of container-1 should apply to the paragraph, and be contained in a simple container with margin on top
- container-1 should be rename to a more descriptive class like .guide-link or something
- it's a matter of opinion or style but I think it does not stand out. I tried with white background. I am not convinced 100%. Other colors are kind of taken. I also tried to incorporate it in the header but I like your idea of like a kind of notification.

![image](https://github.com/Contributees/First-Ruby-Quest/assets/45850166/a8930c07-c360-4522-beea-d7549685ef43)